src/containers/AdyenInitialPayment/AdyenInitialPayment.tsx

```diff
@@ -20,17 +21,20 @@ type Props = {
 };
 
 export default function AdyenInitialPayment({ setUpdatingOrder, type, setPaymentError, paymentSuccessUrl, orderId }: Props) {
+  const accountController = getModule(AccountController);
+  const checkoutController = getModule(CheckoutController);
+
   const [session, setSession] = useState<AdyenPaymentSession>();
 
-  const { sandbox } = useClientIntegration();
+  const { useSandbox } = useConfigStore((s) => s.getIntegration());
```

Comment:
Probably safer to avoid const naming that look like hooks

Reply:
I agree, but we also already use `useSandbox` prop in the config (`integrations` object).

Reply:
Changed to `isSandbox`